### PR TITLE
JNG-4546 Release eclipse-emfatic

### PR DESCRIPTION
**Pull request by gaborflorian**
<table>
<td>
  <a href="https://blackbelt.atlassian.net/browse/JNG-4546" title="JNG-4546" target="_blank">Sub-taskJNG-4546</a>  Release eclipse-emfatic
  </td></table>
  <br />

### CODE REVIEW

File: .github/workflows/build-dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+      - name: ⬇️ Checkout repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+
+      - name: 🛠️ Project context
+        id: context
+        uses: zero88/gh-project-context@v1.2
+
+      - name: 🛠️ java-version: '11'
```
Comment (noherczeg) [inline .github/workflows/build-dependabot.yml]: 17

**Review by noherczeg — CHANGES_REQUESTED** [review]
Missing licensefile

**Review by bencegelei — APPROVED** [review]

File: .github/workflows/build-dependabot.yml (hunk 1 above)
Comment (gaborflorian) [reply .github/workflows/build-dependabot.yml]: [47c3062](https://github.com/BlackBeltTechnology/eclipse-emfatic/pull/31/commits/47c30628e8703d9c896b6b116611e2d685266d7e)

File: pom.xml [hunk 2]
```diff
@@ -399,5 +524,152 @@
...
+                        </configuration>
+                        <executions>
+                            <execution>
+                                <id>first</id>
+                                <goals>
+                                    <goal>update-file-header</goal>
+                                </goals>
+                                <phase>process-sources</phase>
+                                <configuration>
+                                    <licenseName>apache_v2</licenseName>
```
Comment (noherczeg) [inline pom.xml]: Should be epl, because its an eclipse project

**Review by noherczeg — CHANGES_REQUESTED** [review]
Missing license file still.

File: pom.xml (hunk 2 above)
Comment (gaborflorian) [reply pom.xml]: [e33d6f4](https://github.com/BlackBeltTechnology/eclipse-emfatic/pull/31/commits/e33d6f4154a2227a2f16cee2ec4e730cc217eb61)

**Review by bencegelei — APPROVED** [review]

File: pom.xml (hunk 2 above)
Comment (gaborflorian) [reply pom.xml]: https://github.com/BlackBeltTechnology/eclipse-emfatic/pull/31/commits/852c9c319c7641008e9da74e4a4dd35e57f6eba0

**Review by bencegelei — APPROVED** [review]

**Review by robertcsakany — APPROVED** [review]

**Review by noherczeg — APPROVED** [review]

**Pull request merged**